js/TrickleButtonModel.js

```diff
@@ -55,6 +61,15 @@ export default class TrickleButtonModel extends ComponentModel {
     });
   }
 
+  /**
+   * @returns {boolean} true if the parent container is already complete
+   */
+  isStepComplete() {
+    const completionAttribute = getCompletionAttribute();
+    const isParentComplete = this.getParent().get(completionAttribute);
```

Comment:
you could `return this.getParent().get(completionAttribute);` instead of declaring a const and returning the const.

Reply:
It's mainly just for debugging purposes that it's like this atm.
Easier to breakpoint.